metric_info.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env python
+
+metric_default = [
```

Comment:
All metrics are gauges. I have double checked this. You may check it too in file sample_responses.py.